### ISSUE TITLE
Issues with 2.3.0 or 2.3.1 : Build failed when importing SCSS file via JS

### ISSUE THREAD

**Issue by contactjavas** [parcel-bundler/parcel#7690]
# 🐛 bug report

Hi! I've just updated my packages in some minutes / few hours ago. The build is failed when I tried to build a JS file that import an SCSS file.
I see the Parcel version is `2.3.0` (or `2.3.1`). It was fine when it was in `v2.2.1`

## Error messages on the CLI
```bash
Error: mod[method] is not a function

  TypeError: mod[method] is not a function
  at WorkerFarm.processRequest (E:\www\mapsteps\wp-content\plugins\kirki-dev\packages\kirki-framework\control-react-select\node_modules\@parcel\workers\lib\WorkerFarm.js:336:55)
  at Worker.<anonymous> (E:\www\mapsteps\wp-content\plugins\kirki-dev\packages\kirki-framework\control-react-select\node_modules\@parcel\workers\lib\WorkerFarm.js:233:39)
  at Worker.emit (node:events:520:28)
  at Worker.receive (E:\www\mapsteps\wp-content\plugins\kirki-dev\packages\kirki-framework\control-react-select\node_modules\@parcel\workers\lib\Worker.js:177:12)
  at ThreadsWorker.onMessage (E:\www\mapsteps\wp-content\plugins\kirki-dev\packages\kirki-framework\control-react-select\node_modules\@parcel\workers\lib\Worker.js:98:64)
  at ThreadsWorker.handleMessage (E:\www\mapsteps\wp-content\plugins\kirki-dev\packages\kirki-framework\control-react-select\node_modules\@parcel\workers\lib\threads\ThreadsWorker.js:70:10)       
  at Worker.<anonymous> (E:\www\mapsteps\wp-content\plugins\kirki-dev\packages\kirki-framework\control-react-select\node_modules\@parcel\workers\lib\threads\ThreadsWorker.js:55:44)
  at Worker.emit (node:events:520:28)
  at MessagePort.<anonymous> (node:internal/worker:232:53)
  at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:647:20)
```

### package.json
```json
{
    "name": "kirki-control-react-select",
    "author": "Kirki Framework",
    "homepage": "https://kirki.org",
    "license": "MIT",
    "version": "1.1.5",
    "dependencies": {
        "react": "^17.0.2",
        "react-dom": "^17.0.2",
        "react-select": "^5.2.2"
    },
    "devDependencies": {
        "@babel/preset-react": "^7.16.7",
        "parcel": "^2.3.0"
    },
    "alias": {
        "jquery": {
            "global": "jQuery"
        },
        "wp": {
            "global": "wp"
        },
        "react": {
            "global": "React"
        },
        "react-dom": {
            "global": "ReactDOM"
        }
    },
    "browserslist": "> 0.5%, last 2 versions, not dead",
    "source": "src/control.js",
    "scripts": {
        "build": "parcel build",
        "dev": "parcel build --no-optimize",
        "watch": "parcel watch"
    }
}

```

### .babelrc
```json
{
  "presets": [
    [
      "@babel/preset-react",
      {
        "runtime": "classic"
      }
    ]
  ]
}
```

### Example of JS file which build is _failed_
```js
import "./control.scss";
```

### Example of JS file which build is _success_
```js
console.log("when there's no scss import, it's success.");
```

### CLI command to run
`npm run build`

## 🤔 Expected Behavior

When it's still in `v2.2.1` it works fine.

## 😯 Current Behavior

Now it shows error messages (pasted above).

## 💁 Possible Solution
💁

## 💻 Code Sample

Already pasted the package.json & .babelrc content above.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.3.0 or 2.3.1
| Node             | 17.4.0
| npm/Yarn         | 8.3.1
| Operating System | Windows 11

**Comment by devongovett** [comment]
Looks like a duplicate of #7681. Are you sure you have 2.3.1 of all parcel packages?

**Comment by contactjavas** [comment]
Hi @devongovett , thanks for replying :)
Yes, seems like the same / similar issue with #7681 

Here, it works now. Not sure if this is related, some of what I did:
- delete _node_modules_ & _package-lock.json_
- go back to `parcel@2.2.1`
- delete _node_modules_ & _package-lock.json_
- then install `2.3.1` directly

But the build of React is still using the modern runtime even though I set it to classic in _.babelrc_.
But yea, that might be different story.

Let me close this :)
Thanks!
